web-local/src/lib/components/stores/local-storage.ts

```diff
@@ -0,0 +1,37 @@
+import { browser } from "$app/environment";
+import { writable } from "svelte/store";
+
+/** Creates a store whose value is stored in localStorage as a string.
+ * Only JSON-serializable values can be used.
+ */
+export function localStorageStore<T>(defaultValue: T, itemKey: string) {
```

Comment:
This would be easier to read if it were `(key, value)`, rather than `(value, key)`